x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_item_content/relevance_tuning_item_content.scss

```diff
@@ -0,0 +1,6 @@
+.relevanceTuningForm {
+  .relevanceTuningForm__itemContent {
+    border: none;
+    border-top: $euiBorderThin;
+  }
+}
```

Comment:
I'm tempted to move this to `relevance_tuning_form.scss`, since this is a fairly small file that feels like it could easily belongs there, and it's not the worst thing in the world to put all our overrides in the same file

Reply:
You have a valid point. Having the CSS scoped neatly by having a 1 to 1 css to component setup can just make the whole thing feel more manageable. If you ever delete or move relevance_tuning_item_content.tsx, it's super apparent what CSS goes with it.